src/Plugin/RulesAction/DataListItemAdd.php

```diff
@@ -44,16 +44,25 @@
 class DataListItemAdd extends RulesActionBase {
 
   /**
-   * {@inheritdoc}
+   * Executes the Plugin.
+   *
+   * @param array $list
+   *   An array to add an item to.
+   * @param mixed $item
+   *   An item which is being added to the array.
+   * @param bool $unique
+   *   (optional) A boolean value which indicates if the item should be updated
+   *   if it is already in the list array.
+   * @param string $pos
+   *   (optional) Position in array where the item is being added. Accepted
+   *   value are:
+   *   - "start"      Beginning of the array.
```

Comment:
can we use a colon after the value example similar to https://www.drupal.org/coding-standards/docs#lists and without the additional spacing?